nvc/types.go

```diff
@@ -0,0 +1,19 @@
+package nvc
+
+type EntryFlags uint32
+
+const (
+	UNCOMPRESSED EntryFlags = 0
+	COMPRESSED   EntryFlags = 1
+	ENCRYPTED    EntryFlags = 2
+
+	MAGIC = "nvc1d\x00\x00\x00"
```

Comment:
These names should just be lowercase  
(remember that identifiers that start with an uppercase letter are exported; these don't need to be)

Reply:
i saw some conficting style recommendations, what is the standard ?